webdriver/tests/perform_actions/pointer_tripleclick.py

```diff
@@ -0,0 +1,31 @@
+from tests.perform_actions.support.refine import filter_dict, get_events
+from tests.support.asserts import assert_move_to_coordinates
+from tests.support.inline import inline
+
+lots_of_text = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor "\
+               "incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud "\
+               " exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+
+
+def test_tripleclick_at_coordinates(session, mouse_chain):
+    session.url = inline("""<div>
+          {}
+        </div>""".format(lots_of_text))
+    div = session.find.css("div", all=False)
+    div_rect = div.rect
+    div_centre = {
+        "x": div_rect["x"] + div_rect["width"]/2,
+        "y": div_rect["y"] + div_rect["height"]/2
+    }
+    mouse_chain \
+        .pointer_move(div_centre["x"], div_centre["y"]) \
+        .click() \
+        .pause(0) \
+        .click() \
+        .pause(0) \
+        .click() \
+        .perform()
+
+    actual_text = session.execute_script("return document.getSelection().toString();")
```

Comment:
This behaviour seems platform-specific. I'm not sure how else to test though; it seems like double-click has its own event but not triple.

Reply:
From what I can find, all browsers have gone to triple-click means select a paragraph. I can't find anything that suggests it is platform specific. And yes, there is no event I can watch for.

Reply:
What about mobile? At least triple-tap doesn't seem to work the same there. So I think there should at least be a comment indicating that the test might be making nonstandard assumptions to help out people who are investigating failures in the future.

Reply:
comment added